client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import scss from './styles.scss';
```

Comment:
Agreeing with @kpuputti 's earlier comment, should be `css`. The preprocessor we may or may not use shouldn't affect this variable name here.